internal/factory/container/container.go

```diff
@@ -73,8 +73,8 @@ type Container interface {
 	// DisableFips returns whether the container should disable fips mode
 	DisableFips() bool
 
-	// Image returns the image specified in the container spec, or an error
-	Image() (string, error)
+	// UserRequestedHeuristicImage returns the image specified in the container spec, or an error
```

Comment:
similarly here. While i can see a case for being clearer, I think the variable name is very verbose and could be shortened

Reply:
maybe `UserRequestedImage` ?

Reply:
My thinking was that scary concepts should have scary names, to discourage future readers from adding any more uses… and even perhaps motivate (… someone else …) to replace these values entirely.

But really 90% of the value of the exercise for me is in having found/tracked those values, so I’m happy with any names the maintainers prefer.

Reply:
Changed to `UserRequestedImage`.